Packages/nadena.dev.modular-avatar/Runtime/ModularAvatarObjectRemover.cs

```diff
@@ -0,0 +1,52 @@
+﻿using UnityEditor;
+using UnityEngine;
+
+namespace nadena.dev.modular_avatar.core
+{
+    [ExecuteInEditMode]
+    [DisallowMultipleComponent]
+    [AddComponentMenu("Modular Avatar/MA Object Remover")]
+    public class ModularAvatarObjectRemover : AvatarTagComponent
+    {
+        public bool hideInHierarchy;
+
+        public ModularAvatarObjectRemover()
```

Comment:
This should probably be in OnValidate or Awake

Reply:
This is a workaround for unity issue sadly, I need this to get called even if the object is disabled, because unity strips HideFlags from prefab instances on save, and thats the only way I found to restore them

Reply:
Ah, okay. Please add a comment describing the need for this workaround in this case.